ci/main.py

```diff
@@ -118,14 +126,11 @@ def upload(name, version, channel):
         config.bldpkgs_dir,
         '{0}-{1}*.tar.bz2'.format(name, version))
     built = glob.glob(built_glob)[0]
-    upload_cmd = 'anaconda -t {token} upload -u {channel} {built}'
+    upload_cmd = 'anaconda -t {token} upload -u %s %s' % (channel, built)
     # Do not show decrypted token!
     log.info('Uploading: {0}'.format(upload_cmd))
     check_call(
-        upload_cmd.format(
-            token=os.environ['ANACONDA_TOKEN'],
-            built=built,
-            channel=channel),
```

Comment:
I agree that making that change was gilding the lily (cool expression btw) and that's why I was asking to undo, but it's not a big deal and will merge nevertheless.